content/docs/04-clusters/02-data-center/01-maas/10-register-manage-maas-cloud-accounts.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Register and Manage MAAS Cloud Accounts"
+metaTitle: "Register and Manage MAAS Cloud Accounts"
+metaDescription: "Learn how to register and manage your MAAS cloud accounts in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+When you install the Private Cloud Gateway (PCG), a cloud account is auto-created in every project in Palette. You can use this cloud account to create clusters at either the tenant or the project level. If desired, you can create additional cloud accounts that reference specific PCGs. 
+
+# Prerequisites
```

Comment:
Having a MAAS API key is also required. So we can reuse the text we used in the PCG page.

Reply:
Added this prereq to cluster accounts doc.